public/app/features/panel/components/VizTypePicker/VisualizationSuggestionCard.tsx

```diff
@@ -88,13 +96,34 @@ const getStyles = (theme: GrafanaTheme2) => {
         background: ${theme.colors.background.secondary};
       }
     `,
+    imgBox: css`
+      display: flex;
+      flex-direction: column;
+      height: 100%;
+
+      justify-self: center;
+      color: ${theme.colors.text.primary};
+      width: 100%;
+      display: flex;
+
+      justify-content: center;
+      align-items: center;
+      text-align: center;
+    `,
     name: css`
+      padding: ${theme.spacing(1)};
       font-size: ${theme.typography.bodySmall.fontSize};
       white-space: nowrap;
       overflow: hidden;
       color: ${theme.colors.text.secondary};
       font-weight: ${theme.typography.fontWeightMedium};
       text-overflow: ellipsis;
+      padding-bottom: ${theme.spacing(0.5)};
+      margin-top: ${theme.spacing(-2)};
+    `,
+    img: css`
+      max-width: 65px;
+      max-height: 65px;
```

Comment:
should this go in the theme somewhere or can we use a theme value here? either `theme.components.img.sm/md/lg` or `theme.spacing(8)`?